google-finance/package.json

```diff
@@ -1,15 +1,45 @@
 {
   "name": "@chainlink/google-finance-adapter",
   "version": "0.0.1",
+  "description": "Chainlink google-finance adapter.",
+  "keywords": [
+    "Chainlink",
+    "LINK",
+    "blockchain",
+    "oracle",
+    "google-finance"
+  ],
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "repository": {
+    "url": "https://github.com/smartcontractkit/external-adapters-js",
+    "type": "git"
+  },
   "license": "MIT",
-  "main": "index.js",
   "scripts": {
-    "server": "node -e 'require(\"./index.js\").server()'",
```

Comment:
`yarn server` is used by Docker for running the container, so we should keep it.

Reply:
But `./index.js` exists no more. Maybe rename `server:dist` to `server`? This applies to other places as well.

Reply:
Well, `./index.js` exists after the Docker build inside the container. But let's ask @krebernisak on this cause I'm not sure